addons/stock_account/models/account_move.py

```diff
@@ -12,11 +12,9 @@ class AccountMove(models.Model):
     def _compute_show_reset_to_draft_button(self):
         super()._compute_show_reset_to_draft_button()
         for move in self:
-            for line in move.line_ids:
-                # if a line has correction layers hide the 'Reset to Darft' button
-                if line.sudo()._get_stock_valuation_layers(move).stock_valuation_layer_ids.filtered('account_move_line_id'):
-                    move.show_reset_to_draft_button = False
-                    break
+            # TODO justifier (CN qui annule pdiff)
+            if move.sudo().line_ids.stock_valuation_layer_ids:
```

Comment:
:suspect:

Reply:
Yeeeeaaaah I know
It's just a reminder for my commit redaction haha